template/src/client/App.js

```diff
@@ -1,6 +1,5 @@
 import React, { Component } from 'react';
 import logo from './logo.svg';
-import './App.css';
```

Comment:
That's obvious drawback, we can't have the css file directly in the Component file since  `style-loader` simply doesn't work on server. We could use an isomorphic version though.